airflow/contrib/example_dags/example_gcs_to_bq_operator.py

```diff
@@ -28,7 +28,7 @@
 
 gcs_to_bq = None  # type: Any
 try:
-    from airflow.contrib.operators import gcs_to_bq
+    pass
```

Comment:
Can we remove this?

Reply:
Good catch! I also moved this example to core.